rdrf/rdrf/migrations/0136_new_patient_added_data_migration.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.25 on 2022-01-18 18:38
+
+import logging
+
+from django.db import migrations
+
+from ..events.events import EventType
+
+
+logger = logging.getLogger(__name__)
+
+
+def replicate_new_patient_notifications(apps, schema_editor):
```

Comment:
The migration to preserve functionality will create an extra notification in Angelman that should be cleaned up. I think it's better to only duplicate notifications in registries where `PATIENTS_CREATE_USERS` is enabled, or make it MND-specific instead.